.travis.yml

```diff
@@ -12,6 +12,8 @@ matrix:
       env: COLLECT_COVERAGE=true VALIDATE_CODING_STYLE=false
     - php: 7.1
       env: COLLECT_COVERAGE=true VALIDATE_CODING_STYLE=false
+    - php: 7.2
+      env: COLLECT_COVERAGE=true VALIDATE_CODING_STYLE=false
```

Comment:
I'll accept this modification but I'll update it before release to tune it a little bit